subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/fingerprint/ConfigurationCacheFingerprintWriter.kt.orig

```diff
@@ -0,0 +1,669 @@
+/*
```

Comment:
I guess you didn't mean this file to be included :)

Reply:
Oops. Forgot to configure my global `.gitignore`. Will remove in the followup CL.